### PR TITLE
Added New Nav Link to View Other States

### PR DESCRIPTION
**Pull request by AshleyMSherwood**
Closes issue #52. I think this satisfied the issue, but this is my first time contributing, so please let me know if anyone has any feedback. Thanks!

### CODE REVIEW

**Review by sigmavirus24 — APPROVED** [review]
I know next to nothing about Nuxt but this looks good to me and works locally.

**Comment by GUI** [comment]
Thank you! This week has gotten the best of me, so sorry I'm slow in reviewing and merging this (among other things), but I'm really trying to get to all these PRs this week. But in the meantime, just wanted to say thank you!

File: website/components/Navbar.vue [hunk 1]
```diff
@@ -37,6 +42,11 @@ export default {
       },
     };
   },
+  computed: {
+    routeURL() {
+      return this.$route.name !== "index";
+    },
```
Comment (Maker-Mark) [inline website/components/Navbar.vue:48]: We may want to rename this to "showStateNav" or something similar so it's obvious that this is a boolean. I was just working on this, and then I saw you made it!

**Comment by GUI** [comment]
@AshleyMSherwood: My apologies for letting this languish so long! After mulling this over some more and hearing other use-cases, I've implemented a slightly different workaround in https://github.com/GUI/covid-vaccine-spotter/commit/d91cd1ddb07d6b9413284508bbe7ca3784e2054f, which adds a select menu to the navbar. My thought with this approach is that it should allow for quicker switching directly between states, but let me know if you have any other thoughts or feedback. Thank you!

**Pull request closed without merging**